admin/components/AliasedLinkTable.tsx

```diff
@@ -130,6 +138,12 @@ export default function AliasedLinkTable(props: AliasedLinkTableProps) {
               aliasedLink.public === value) as any
           }
         />
+        <Column
+          title="Hits"
+          dataIndex="hits"
+          key="hits"
+          sorter={(a: AliasedLinkType, b: AliasedLinkType) => a.hits - b.hits}
+        />
```

Comment:
I cant see the hits since I keep getting routed to production deploy after auth, but I trust this'll be good to go :)

Reply:
Yeah the big limitation with the auth here is that I can't programmatically redirect back to the environment in question, but I tested locally and it does seem to work